testcafe/VisualReview-node-client/index.js

```diff
@@ -1,130 +1,142 @@
-import request from 'request';
-import fs from 'fs';
-import url from 'url';
-import { EOL } from 'os';
-import defaultOptions from './defaultOptions';
+const request = require('request')
+const fs = require('fs')
+const url = require('url')
+const EOL = require('os')
+const defaultOptions = require('./defaultOptions')
```

Comment:
I change from 'import' to 'require' because as I first try to add visualReview as a node module I was getting errors using `import`.
If we don't re-create a node module and just use visualReview as an 'helper`, there's no need to change to 'require' so I can change it back!